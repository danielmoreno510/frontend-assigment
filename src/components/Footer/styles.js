const styles = {
  container: {
    background: "#1b80c6",
    display: "flex",
    color: "white",
    justifyContent: "space-between",
    paddingRight: "32px",
    fontSize: "14px"
  },
  sectionsFlex: {
    display: "flex",
    alignItems: "center",
  },
  sectionsFlexCopyright: {
    display: "flex",
    alignItems: "center",
    color: "#ffffff75",
  },
  containerLogo: {
    padding: "12px 0px 12px 32px",
  },
  logo: {
    height: "44px",
  },
};

export default styles;
