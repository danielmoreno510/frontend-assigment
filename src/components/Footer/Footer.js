import React from "react";

import styles from "./styles";
import logo from "../../assets/images/logo.png";

const Footer = () => {
  return (
    <div style={styles.container}>
      <div style={styles.containerLogo}>
        <img style={styles.logo} src={logo} />
      </div>
      <div style={styles.sectionsFlex}>Preguntas Frecuentes</div>
      <div style={styles.sectionsFlex}>Contáctos</div>
      <div style={styles.sectionsFlex}>Información</div>
      <div style={styles.sectionsFlexCopyright}>
        DocRed © 2019. Todos los derechos reservados
      </div>
    </div>
  );
};

export default Footer;
