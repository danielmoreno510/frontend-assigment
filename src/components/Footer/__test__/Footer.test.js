import React from "react";
import Adapter from "enzyme-adapter-react-16";
import { shallow, configure } from "enzyme";

import Footer from "../Footer";

configure({ adapter: new Adapter() });
const testProps = {
  articles: [],
};

describe("Footer component", () => {
  describe("WHEN the component is render", () => {
    const wrapper = shallow(<Footer {...testProps} />);

    it("THEN should display Footer component", () => {
      expect(wrapper).toMatchSnapshot();
    });
  });
});
