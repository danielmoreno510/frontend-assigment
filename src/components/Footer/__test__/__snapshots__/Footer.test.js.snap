// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component WHEN the component is render THEN should display Footer component 1`] = `
<div
  style={
    Object {
      "background": "#1b80c6",
      "color": "white",
      "display": "flex",
      "fontSize": "14px",
      "justifyContent": "space-between",
      "paddingRight": "32px",
    }
  }
>
  <div
    style={
      Object {
        "padding": "12px 0px 12px 32px",
      }
    }
  >
    <img
      src="logo.png"
      style={
        Object {
          "height": "44px",
        }
      }
    />
  </div>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
      }
    }
  >
    Preguntas Frecuentes
  </div>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
      }
    }
  >
    Contáctos
  </div>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
      }
    }
  >
    Información
  </div>
  <div
    style={
      Object {
        "alignItems": "center",
        "color": "#ffffff75",
        "display": "flex",
      }
    }
  >
    DocRed © 2019. Todos los derechos reservados
  </div>
</div>
`;
